pages/linux/timeout.md

```diff
@@ -0,0 +1,11 @@
+# timeout
+
+> Run a command with a time limit.
+
+- Run sleep command (for 10 seconds) but kill it after 3 seconds:
+
+`timeout 3s sleep 10`
+
+- Run sleep command for 10 seconds but kill it after 3 seconds or when interruptted:
```

Comment:
Perhaps putting `for 10 seconds` in brackets like the first example would improve consistency?

And what's the difference between this command and the last?

Reply:
I agree. Adding the brackets will improve the consistency. The last command add the usage case for --signal INT on interrupt signal case. Either timeout or an interrupt signal will kill the process. In the first command, only timeout will kill the process. Both of them are common use case.